Tests.HealthCheck/EmailSender.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using HealthChecks.UI.Core;
+using Tests.HealthCheck.Models;
+
+namespace Tests.HealthCheck
+{
+    public static class EmailSender
+    {
+        private static readonly List<string> _reports = new();
+        private static TimeSpan _interval;
+        private static string[] _emails;
+        private static SmtpCredentialsOptions _options;
+        
+        public static void AddReport(UIHealthReport newReport)
+        {
+            _reports.Add(string.Join(
+                ", ", 
+                newReport.Entries.Values.Select(x => x.Description)));
+        }
+
+        public static void Start(int interval, string[] emails, SmtpCredentialsOptions options)
```

Comment:
whose emails are these ?

Reply:
It is from app settings, there is only one my email, in future, there will be Spartak's email